.github/workflows/pull-docker_dryrun.yml

```diff
@@ -0,0 +1,19 @@
+name: Docker build dry run
+
+on: [pull_request]
+
+jobs:
+  docker_dryrun:
+    runs-on: ubuntu-latest
```

Comment:
It does not actually run on ARM anymore like previous pipeline did, right? Can GitHub Actions do ARM?

Reply:
According to https://github.com/orgs/community/discussions/25319, no, arm is not officially supported, only through self-hosted runner it seems.

Reply:
Good catch, I was thinking I saw before that it was possible but I must have misremembered.

Reply:
If we completely replace drone, I guess it would be best to run [github's runner](https://github.com/actions/runner), both from a resource and architecture support perspective.